### ISSUE TITLE
issue about model.py/dropout = tf.layers.dropout(pool, rate=drop_rate)

### ISSUE THREAD

**Issue by YanShuang17** [potterhsu/SVHNClassifier#16]
Hello, when i train the model with my own dataset,  i found the statement:
`dropout = tf.layers.dropout(pool, rate=drop_rate)`
in model.py may be have problem.
The operation: **tf.layers.dropout()**  in TensorFlow's document has an argument of training, and **training=False** by default, in your code, you didn't set the **training** argument, so, although you set the **rate** value, the model won't conduct the dropout operation???
